packages/app-project/src/screens/ProjectHomePage/components/Hero/Hero.spec.js

```diff
@@ -18,14 +18,14 @@ describe('Component > Hero', function () {
   })
 
   it('should render the `Background` component', function () {
-    expect(wrapper.find(Background)).to.have.lengthOf(1)
+    expect(wrapper.find(Background)).to.have.lengthOf(2)
```

Comment:
This would be better if it could test the two layouts separately, and expect each layout to have a single background, introduction and workflow selector.

Reply:
Maybe find the `Media` components by their `at` props and run tests on each for background, introduction and workflow selector? Something like tests for linked media here:
https://github.com/zooniverse/front-end-monorepo/blob/757a9c48efd75e7ea11af147304b751b603882e7/packages/app-project/src/screens/ProjectHomePage/components/ZooniverseTalk/components/RecentSubjects/components/SubjectThumbnail/SubjectThumbnail.spec.js#L30-L35